lib/CheckboxGroup.js

```diff
@@ -0,0 +1,43 @@
+"use strict";
+var __assign = (this && this.__assign) || function () {
+    __assign = Object.assign || function(t) {
+        for (var s, i = 1, n = arguments.length; i < n; i++) {
+            s = arguments[i];
+            for (var p in s) if (Object.prototype.hasOwnProperty.call(s, p))
+                t[p] = s[p];
```

Comment:
can we wrap single line for and if statements with { } for readability?

Reply:
I edited this pull request a bit late to explain the code.  This is generated code.